src/components/Resources/ResourcePage.js

```diff
@@ -3,8 +3,16 @@ import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
 import axios from 'axios';
 import PersonalMenu from '../PersonalMenu';
-import { Grid, Breadcrumbs, Typography } from '@material-ui/core';
+import { Grid, Breadcrumbs, Typography, Chip } from '@material-ui/core';
 import NavigateNextIcon from '@material-ui/icons/NavigateNext';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+  subtitle: {
+    display: 'flex',
+    justifyContent: 'space-between',
+  },
+});
 
 function ResourcePage({ matchProps }) {
   const [resource, setResource] = useState({});
```

Comment:
@bengineerdavis @lpatmo - I'm finding it kind of a smell to using `useState` to save data here. Any thoughts on other alternatives for handling this? `resource` isn't really a state 🤔 

I think you can use Axios to fetch data and assign it to a variable and render that down.

Reply:
> I think you can use Axios to fetch data and assign it to a variable and render that down.

Hmm... I think that will only work if we have a custom hook like a `useFetch` that returns something like a value, loading state, and error state from the request. Then we'd just use the `value` returned in this component and render that down. 

@angelocordon Can you share an example of what you mean? ` const resource = response.data;` will be scoped to the `.then` and won't be available down below.

Reply:
I'm with however you and @lpatmo want to handle this.

Reply:
I'm researching more into this, but I'm pretty sure you can just assign this to a variable and render that variable. 

```javascript
const Component = () => {
    const data = axios.get(`url/${params.guid}`);

    return (
        <span>{ data }</span>
    )
}
```

Reply:
Hmm... the array of resources isn't available until we hit the `.then` in the ```axios
      .get('/api/v1/resources/' + matchProps.match.params.guid)
      .then(function(response) {``` promise, so I think we need to keep that `.then` and the `.catch`. 

But there's a pattern where we can create a custom hook (e.g. `useFetch`) that does the axios calls for us, so that we can do something like ` const data = useFetch(`url/${params.guid}`);` and reuse that later.

Reply:
@angelocordon Let me know if you also want to (or have a bit of time) to pair on this!

Reply:
@lpatmo some folks recommended this library and I think it fits what we want here https://react-query.tanstack.com/docs/overview